billiard/process.py

```diff
@@ -270,7 +263,7 @@ def _bootstrap(self):
                 except (OSError, ValueError):
                     pass
             old_process = _current_process
-            _set_current_process(self)
+            _current_process = self
```

Comment:
Don't we already have logging info set without directly setting in the multiprocessing lib?
Perhaps I'm naiive, but I have been getting log info set and coming out with no problems (with these patches) while multiprocessing has not even been loaded (i.e. does not exist in sys.modules at exit).